### DISCUSSION TITLE
2008/07/01/%E5%87%86%E5%A4%87%E6%8A%8A-blog-%E4%B8%8A%E7%9A%84%E6%96%87%E7%AB%A0%E6%95%B4%E7%90%86%E6%88%90%E4%B9%A6

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#226] [Announcements]
# 2008/07/01/%E5%87%86%E5%A4%87%E6%8A%8A-blog-%E4%B8%8A%E7%9A%84%E6%96%87%E7%AB%A0%E6%95%B4%E7%90%86%E6%88%90%E4%B9%A6

最近跟北航出版社的一位老师联系了几次，在他的建议和鼓励下，开始正式考虑把LabVIEW相关的内容编辑成书。

https://blog.qizhen.xyz/2008/07/01/%E5%87%86%E5%A4%87%E6%8A%8A-blog-%E4%B8%8A%E7%9A%84%E6%96%87%E7%AB%A0%E6%95%B4%E7%90%86%E6%88%90%E4%B9%A6

**Comment by ruanqizhen** [comment]
之前博客下的留言：

秋
2008年12月23日 10:40 编辑
我也有过这种想法~可是有时候暗自觉得不太现实~博客坚持了也有三年之久了~应该是本很厚的书罢~~~~~

回复
Jucao
2008年12月17日 15:10 编辑
期待你的大作问世，呵呵。

回复
半夏
2008年12月16日 08:15 编辑
很好的想法加油

回复
Qizhen
2008年12月15日 16:00 编辑
我只能尽量认真的写，但是我的水平和精力也是有限的，可能没有大家期望的那么高。

回复
Unknown
2008年12月8日 11:27 编辑
"在技术杂志上发表文章的主要都是为了取得学位或职称的" "中国很多技术书籍以抄袭为主，甚至不是作者自己抄，而是找几个学生代工。把国外相关书籍各处翻译一下，东拼西凑就成了一本书。" 对此大家太有同感了! 衷心希望阮先生写出一本切合实际的Labview教材,造福广大的Labview爱好者. 一点建议:毕竟初学者居多数，不要一开始就教得很深，拒初学者于门外。可以分上中下册出版。“入门”/“提高”/“高级”……分期出版，这样时间上不至于太紧张。 对于你的书，大家那是相当期待，都圪那憋着呢！

回复
Hongbo
2008年09月9日 21:41 编辑
小弟潜水了很久，从阮大侠的LabVIEW文章中收益匪浅，也提点建议。主要想法是，即将要出的这本书就应该定位给中级以上用户。俺不敢宣称自己是高级用户，但俺觉得有一个很大的群体像俺一样，通过自学或培训LabVIEW之后参加和完成了有一定规模的项目，需要补充高级技巧来完善和提高当前或将来的项目。可以想象在座各位对于国内那些翻译抄袭LabVIEW手册的书也大都不屑一顾。因此俺建议阮大侠只要把博客上这些内容简单编辑整理一下，用不了六个月，半个月就足够了。一本好书贵不在厚，而在精。书名就叫“LabVIEW高级编程技巧选谈”。以后还可补充更新出第二版…第N版，乃至英文版（按楼上说的，借NI大旗用）。愚者拙见，还望指教！

回复
hybetter
2008年07月8日 11:05 编辑
期待着阮大侠的新作！

回复
Nicholas
2008年07月5日 11:51 编辑
不管畅销不畅销，在座的怎么也得每人送一本吧？

回复
昊
2008年07月3日 14:42 编辑
“LabVIEW 的用户几乎还全部是初级用户”
是啊，我毕业半年，做LabVIEW差不多一年了，做了至少两个项目（一个航天领域、一个电力领域），可还是觉得自己是LabVIEW的初级用户。

回复
ian
2008年07月3日 09:45 编辑
Hope to see something that benefits all :) Cheers!
 
Ian.F

回复
agilentvee
2008年07月2日 19:56 编辑
大哥的文章学问味太重，很多并没有谈到实际应用。希望有机会能讲一些解决项目的经验。

回复
Qizhen
2008年07月2日 19:17 编辑
我不是北航的，只不过跟我联系的老师是北航出版社的。我本科东南大学毕业，后来在上海交大读了个工程硕士。后来交大度那个学位其实去学校上课的时间不多，所以心底里我更认同自己是东南大学的。不过周围同事多是交大毕业的，我也经常自称是交大校友，跟着蹭蹭饭:)

回复
Coral
2008年07月2日 13:37 编辑
罗嗦了点，不就是四个字–决定出书嘛。当然应该出了，这样给NI奉献的那些青春也算是有点东西留下，再让NI把你的书作为那些Training课程的推荐书籍，就大卖了。有多的再送我些，我拿去网上卖卖也赚点外快，哇咔咔 :P

回复
xiaoji
2008年07月2日 12:34 编辑
支持！一直见你坚持不懈的写关于技术的东东在blog上，现在有了明确的目标就努力去实现吧。
哪怕就当是个心愿，偿了它！

回复
Angela
2008年07月2日 10:06 编辑
啊？您是北航的？我们是校友吗？

回复
ruirui
2008年07月1日 22:31 编辑
真是个好消息 :)
等到发表的那一天我也要去买,有机会亲自回来找你签名,哈哈

回复
Zhen
2008年07月1日 22:23 编辑
哈哈，太好了，期待您的大作！
我觉得初级的东西大部分都能在help中得到帮助，市面上的很多书籍也基本就是把help翻译过来删删补补，也够参考的了
对我而言，还是希望看到像您blog中的那些文章那样的，涉及高级编程的多一点
LabVIEW现在的应用领域很广了，如果可以介绍一些多领域应用的经验那就更好了
 

回复
